### DISCUSSION TITLE
Error by Running with Wrover IE / Help needed

### DISCUSSION THREAD

**Discussion by Micha2007fe** [schreibfaul1/ESP32-audioI2S#430] [Q&A]
Hi,
I used the project with Wrover IE and it works perfect.
Now I have a Wrover  IB and the project do not works anymore.

IB has the DOWF Chip and the IE the DOWF-V3.

After start the project the ESP with the newer IE reboot with this text:
Curious that first I connect to host and then  the Erros comes.

13:48:37.939 -> rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
13:48:37.939 -> configsip: 0, SPIWP:0xee
13:48:37.939 -> clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
13:48:37.979 -> mode:DIO, clock div:2
13:48:37.979 -> load:0x3fff0018,len:4
13:48:37.979 -> load:0x3fff001c,len:1216
13:48:37.979 -> ho 0 tail 12 room 4
13:48:37.979 -> load:0x40078000,len:10944
13:48:37.979 -> load:0x40080400,len:6360
13:48:37.979 -> entry 0x400806b4
**13:48:39.205 -> Point1info        Connect to new host: "https://bob.hoerradar.de/radiobob-national-mp3-hq"**
1**3:49:01.879 -> Guru Meditation Error: Core  1 panic'ed (LoadProhibited). Exception was unhandled.
13:49:01.879 -> Core 1 register dump:
13:49:01.879 -> PC      : 0x4009145f  PS      : 0x00060e30  A0      : 0x80091a33  A1      : 0x3ffb19b0  
13:49:01.879 -> A2      : 0x3f800000  A3      : 0x000493ec  A4      : 0x00000000  A5      : 0x00000000  
13:49:01.879 -> A6      : 0x00000000  A7      : 0x3ffd7ce5  A8      : 0x00000000  A9      : 0xffffffff  
13:49:01.879 -> A10     : 0x3f800010  A11     : 0x00000000  A12     : 0x3f800014  A13     : 0x00000000  
13:49:01.919 -> A14     : 0xffffffff  A15     : 0x00000000  SAR     : 0x00000005  EXCCAUSE: 0x0000001c  
13:49:01.919 -> EXCVADDR: 0xffffffff  LBEG    : 0x4008c202  LEND    : 0x4008c20d  LCOUNT  : 0x00000000**  
13:49:01.919 -> 
13:49:01.919 -> ELF file SHA256: 0000000000000000
13:49:01.919 -> 
13:49:01.919 -> Backtrace: 0x4009145f:0x3ffb19b0 0x40091a30:0x3ffb19d0 0x400826a9:0x3ffb19f0 0x400828e0:0x3ffb1a10 0x40081d27:0x3ffb1a30 0x400dc5a2:0x3ffb1a50 0x400dc8b9:0x3ffb1cd0 0x400d184c:0x3ffb1f80 0x400eb652:0x3ffb1fb0 0x4008e386:0x3ffb1fd0
13:49:01.959 -> 
13:49:01.959 -> Rebooting..

**Comment by Micha2007fe** [comment]
Ups ...here the correct info :  
Works fine with : IB , DOWD_
Error with:           IE , DOWD_V3

**Comment by Kiwi915** [comment]
Solved
